guide/mechanics/damageFormula.md

```diff
@@ -8,27 +8,35 @@ sidebar_label: Damage Formula
 
 ## The Formula
 
-Anthem's damage formula is as follows:
+It helps to break up the damage formula as follows
+
+```DamageDealt = NormalDamage * (1 + CriticalBonus) + 1```  
+
+where
+
+```NormalDamage = base *(1 +  SumDamageModifiers) * TypeModifier * (1 + DebuffModifiers)```
+
+and
+
+```CriticalBonus = ItemCritBonus * WeakPointMultiplier * (1 + CritModifiers)```
 
-```DAMAGE = (Base* (1+A)*B*C*(1+D)*E*F)+1```
 
 
 ### Parameters used
 
 **Base:** Card damage on the Weapon/Ability. For secondary damage sources this value is hidden.
 
-**A:** Damage Modifiers. If you’re looking at a weapon with 75% weapon damage and no other modifiers, this number is 0.75. However, this should be the sum of every applicable damage modifier. Consult the [Damage Types](mechanics/damageTypes.md) guide for more information about how to calculate this.
-
-**B:** Source Crit Multiplier. For anything that cannot crit, this value is 1. For anything that can crit, this value depends on the damage source. Every weapon and ability that can crit has a crit multiplier specific to the item. Blastback is 2.4, Plasma Star is 1.5. a full list of datamined values can be found on [AnthemArchive](http:AnthemArchive.com) Please note that shielded enemies cannot be critically hit for damage purposes.
+**SumDamageModifiers:** If you’re looking at a weapon with 75% weapon damage and no other modifiers, this number is 0.75. However, this should be the sum of every applicable damage modifier. Consult the [Damage Types](mechanics/damageTypes.md) guide for more information about how to calculate this.
 
-**C:** Target Crit Multiplier For anything that cannot crit, this value is 1. Each weakpoint on an enemy has it's on crit multiplier. For most enemies, this value is 0.5. Some enemies such as scars, outlaw shotgunners, and Ash Titan's have higher or lower values.
+**TypeModifier:** This represents the damage bonus (or penalty) applied based on the damage type of your attack and the health type of the enemy. All attacks fall into one of 5 categories: Impact, Acid, Electric, Ice, or Fire. A table showing what the value of D should be is shown in the table below
```

Comment:
Provide a single common example in this paragraph so people don't necessarily need to go to the table. Like "For example, lightning against shields is `1.5` here".

Reply:
good idea